app/assets/stylesheets/sdg/help.scss

```diff
@@ -0,0 +1,29 @@
+.sdg-help-content {
+  @include grid-column-gutter;
+
+  > .tabs {
+    $spacing: 0.5%;
+
+    li {
+      min-width: $sdg-icon-min-width;
+      width: calc(100% / 17 - #{$spacing});
```

Comment:
This code adds some duplication with the related list selector but does not add the minimum margin rules :thinking:, which is more apparent on mobile phones.

Here's a possible patch:

[sdg_selector.txt](https://github.com/consul/consul/files/6050217/sdg_selector.txt)

Note we need to add the `.tabs-title` selector in order to override rules set by Foundation :relieved:.